plugin/session_view.py

```diff
@@ -127,7 +127,13 @@ def present_diagnostics_async(self, flags: int) -> None:
                 self.view.erase_regions(key)
             elif ((severity <= userprefs().show_diagnostics_severity_level) and
                     (data.icon or flags != (sublime.DRAW_NO_FILL | sublime.DRAW_NO_OUTLINE))):
-                self.view.add_regions(key, data.regions, data.scope, data.icon, flags)
+                def handle_same_regions(region: sublime.Region) -> sublime.Region:
+                    # this allows showing diagnostics with same begin and end in the view
+                    if region.begin() == region.end():
```

Comment:
It's sufficient to check `region.a == region.b` here.

Reply:
I initially put those `a` and `b`,
but then I changed to `begin`, and `end` methods because it looked more readable. 

but I will bring it back to `a` and `b`